pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/stream/ReadableStreamingData.java

```diff
@@ -151,14 +151,16 @@ public byte readByte() {
     /** {@inheritDoc} */
     @Override
     public long skip(final long n) {
-        final long clamped = Math.min(n, limit);
```

Comment:
This behavior is stated in `ReadableSequentialData.skip()`, so its JavaDoc should be updated.

Reply:
It's actually `SequentialData.skip()`. Updated.